print-default.properties

```diff
@@ -122,4 +122,13 @@ mosip.print.crypto.p12.password=password@123
 mosip.print.crypto.p12.alias=partner
 
 # verifiable credential
-mosip.print.verify.credentials.flag=true
+mosip.print.verify.credentials.flag=false
+
+# Object store
+object.store.s3.accesskey=${s3.accesskey}
+object.store.s3.secretkey=${s3.secretkey}
+## For Minio: object.store.s3.url=http://minio.minio:9000
+## For AWS: object.store.s3.url=s3.${s3.region}.amazonaws.com
+object.store.s3.url=http://minio.minio:9000
+object.store.s3.region=${s3.region}
+object.store.s3.readlimit=10000000
```

Comment:
Why do we need to connect to minio / object store from the print service ?

Reply:
devops team is added this we are uploading pdf bytes to datashare

Reply:
As discussed, @dhanendra06 to check with devops team and remove if not required

Reply:
@vishwa-vyom @dhanendra06
while onboarding certs for print-service we need this properties, to push that cert reports into minio

Reply:
Is the onboarder scripts consuming this properties, in that case, can we not keep this in a separate property file ?
cc: @Mahesh-Binayak

Reply:
The onboarder script takes the properties and secrets from config-server directly @vishwa-vyom @dhanendra06 .
So its independent of these values.

Reply:
Thanks for the confirmation @Mahesh-Binayak 
@dhanendra06 can we revert this change also ?

Reply:
reverted the changes @vishwa-vyom